lib/notifier/webhook.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios')
+const _ = require('lodash/fp')
+const uuid = require('uuid')
+
+const sendMessage = (settings, rec) => {
+  const webhookUrl = process.env.WEBHOOK_URL
```

Comment:
Is the URL expected to change ever after program start? Reading envvars can be expensive, so if possible it should be done once only, not on every call.